woo-razorpay.php

```diff
@@ -2029,6 +2029,23 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
 
             $note = __('Order placed through Razorpay Magic Checkout');
             $order->add_order_note( $note );
+            if (!$webhook && $paymentDoneBy === 'cod')
```

Comment:
What if the callback fails and the order is placed through the webhook script, in that case, this conversion API won't trigger?